Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js

```diff
@@ -184,6 +227,17 @@ function vtkResliceCursorContextRepresentation(publicAPI, model) {
         actor.setPickable(state.getEnableTranslation());
       }
     });
+    let lineThickness = state.getLineThickness();
+    if (renderingType === RenderingTypes.PICKING_BUFFER) {
+      lineThickness = Math.max(3, lineThickness);
+    }
+    publicAPI.setLineThickness(lineThickness);
+
+    let radius = state.getSphereRadius();
+    if (renderingType === RenderingTypes.PICKING_BUFFER) {
```

Comment:
Is there a reason to have two `if (renderingType === RenderingTypes.PICKING_BUFFER)` ?
Else, maybe we can factorize:
`
let lineThickness = state.getLineThickness();
let radius = state.getSphereRadius();
if (renderingType === RenderingTypes.PICKING_BUFFER) {
  lineThickness = Math.max(3, lineThickness);
  radius += 1;
}
publicAPI.setLineThickness(lineThickness);
publicAPI.setSphereRadius(radius);
`

Reply:
This is an interesting good point.

In this particular case, for some reason (and I admit I do not have a scientific proof to provide), I find it better (more readable) to keep linethickness separated from sphere radius.

But if you feel strong about it, I can factorize like you suggest.

Reply:
I was just surprised, especially from you, to have e dupplication in your code :)
But it's just a style, you can keep it as you feel.